### PR TITLE
Add git to prerequisites for getting started with F# on vscode article

### PR DESCRIPTION
**Pull request by cartermp**

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/getting-started/getting-started-vscode?branch=pr-en-us-1180](https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/getting-started/getting-started-vscode?branch=pr-en-us-1180)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C27%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610271714488475-1180%5Cworkflow_report.html).

File: docs/fsharp/tutorials/getting-started/getting-started-vscode.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ You can write F# in [Visual Studio Code](https://code.visualstudio.com) with the
 
 F# 4.0 or higher must be installed on your machine to use Ionide.
 
+You must also have [git installed](https://git-scm.com/download) and available on your PATH to make use of project templates in Ionide.  You can verify that it is installed correctly by typing `git` and pressing **Enter** into a command line.
```
Comment (mairaw) [inline docs/fsharp/tutorials/getting-started/getting-started-vscode.md]: nit: from style guide: Customers enter commands at a command prompt, not on a command line.

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/fsharp/tutorials/getting-started/getting-started-vscode.md](https://github.com/cartermp/core-docs/blob/update-vscode-fs/docs/fsharp/tutorials/getting-started/getting-started-vscode.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/getting-started/getting-started-vscode?branch=pr-en-us-1180) | [Details](#user-content-docs/fsharp/tutorials/getting-started/getting-started-vscode.md)

<a name="docs/fsharp/tutorials/getting-started/getting-started-vscode.md"></a>
### [docs/fsharp/tutorials/getting-started/getting-started-vscode.md](https://github.com/cartermp/core-docs/blob/update-vscode-fs/docs/fsharp/tutorials/getting-started/getting-started-vscode.md)
 - **Line 29**: **[Warning]** docs/fsharp/tutorials/getting-started/getting-started-vscode.md contains illegal link: docs/fsharp/tutorials/getting-started/getting-started-visual-studio.md#installing-the-visual-f#-tools. Link text is `Install the Visual F# Tools`. The file docs/fsharp/tutorials/getting-started/getting-started-visual-studio.md doesn&#39;t contain a bookmark named installing-the-visual-f#-tools.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C11%5C12%5Ca3bda507-3390-de91-8f7f-26f90f4e5fc8%5CPullRequest%5C201611121955004318-1180%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/fsharp/tutorials/getting-started/getting-started-vscode.md](https://github.com/cartermp/core-docs/blob/update-vscode-fs/docs/fsharp/tutorials/getting-started/getting-started-vscode.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/getting-started/getting-started-vscode?branch=pr-en-us-1180) | [Details](#user-content-docs/fsharp/tutorials/getting-started/getting-started-vscode.md)


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C11%5C12%5Ca3bda507-3390-de91-8f7f-26f90f4e5fc8%5CPullRequest%5C201611122152416149-1180%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**